files/etc/workarea.env

```diff
@@ -0,0 +1,14 @@
+#
+# Workarea Application Configuration
+#
+# For more configuration settings, check out:
+# https://developer.workarea.com/articles/configuration-for-hosting.html
+
+RAILS_ENV="production"
+WORKAREA_MONGOID_HOST="localhost:27017"
+WORKAREA_ELASTICSEARCH_URL="localhost:9200"
+WORKAREA_REDIS_HOST="localhost"
+WORKAREA_REDIS_PORT="6379"
+WORKAREA_REDIS_CACHE_HOST="localhost"
+WORKAREA_REDIS_CACHE_PORT="6379"
+MALLOC_ARENA_MAX=2
```

Comment:
this is also from the systemd sidekiq service and refers to https://www.mikeperham.com/2018/04/25/taming-rails-memory-bloat/